katalog/velero/README.md

```diff
@@ -0,0 +1,151 @@
+# Velero
+
+![Velero Logo](../../docs/assets/velero.png)
+
+- [Velero](#velero)
+  - [Requirements](#requirements)
+  - [Server deployment](#server-deployment)
+    - [Velero on premises](#velero-on-premises)
+    - [Velero in AWS](#velero-in-aws)
+    - [Velero in GCP](#velero-in-gcp)
+    - [Velero in Azure](#velero-in-azure)
+    - [Velero Restic](#velero-restic)
+
+___
+
+Velero *(formerly Heptio Ark)* gives you tools to back up and restore your Kubernetes cluster resources and persistent
+volumes. You can run Velero with a cloud provider or on-premises. Velero lets you:
+
+- Take backups of your cluster and restore in case of loss.
+- Migrate cluster resources to other clusters.
+- Replicate your production cluster to development and testing clusters.
+
+Velero consists of:
+
+- A server that runs on your cluster
+- A command-line client that runs locally
+
+
+## Requirements
+
+Velero requires to have already deployed the [prometheus-operator](https://github.com/coreos/prometheus-operator) CRDs
+as this feature deploys [a `ServiceMonitor` definition](velero-base/serviceMonitor.yaml). It can be deployed using the
+[fury-kubernetes-monitoring](https://github.com/sighupio/fury-kubernetes-monitoring) KFD core module.
+
+
+## Server deployment
+
+Every velero deployment, does not matter if is on premises or in any of the supported cloud has preconfigured an
+[schedule](velero-base/schedule.yaml) to backup all cluster manifests in an object storage. The backup is triggered
+every 15 minutes.
+
+
+### Velero on premises
+
+The [velero-prem](./velero-prem/) feature deploys a [MinIO](https://min.io/) instance in the same cluster as
```

Comment:
Do we really want to backup the state of the cluster in the cluster itself?

This way, we're just moving the problem around but not actually solving it.

Reply:
At least, this way we provide a way to rollback any kubernetes resource if something was accidentally deleted. "bene ma non benissimo"

Another option is to provide a way (ansible/terraform) to create an external MinIO instance. I don't know if it could be quite overkill. What do you think?